fe1-web/src/features/wallet/screens/WalletHome.tsx

```diff
@@ -72,12 +104,65 @@ const WalletHome = () => {
       });
   }, [rollCalls, laoId]);
 
+  useEffect(() => {
+    if (!tokens || selectedTokenIndex === -1) {
+      setIsSelectedLaoOrganizer(false);
+      return;
+    }
+    const organizer = walletContext.getLaoOrganizer(tokens[selectedTokenIndex].laoId.valueOf());
+    if (organizer && organizer.valueOf() === KeyPairStore.getPublicKey().valueOf()) {
+      setIsSelectedLaoOrganizer(true);
+    } else {
+      setIsSelectedLaoOrganizer(false);
+    }
+  }, [walletContext, tokens, selectedTokenIndex]);
+
+  const sendTransaction = (receiver: string, amount: number, isCoinbase: boolean) => {
+    if (!tokens) {
+      throw new Error('No pop tokens found to send a transaction');
+    }
+    if (selectedTokenIndex < 0) {
+      throw new Error('No pop tokens selected to send a transaction');
+    }
+
+    if (isCoinbase) {
+      requestCoinbaseTransaction(
+        KeyPairStore.get(),
+        [new PublicKey(receiver)],
+        amount,
+        tokens[selectedTokenIndex].laoId,
+      ).then(
+        () => toast.show('Sent coinbase transaction'),
+        (reason) => {
+          toast.show('Failed to send transaction');
+          console.error(reason);
+        },
+      );
+    } else {
+      requestSendTransaction(
+        tokens[selectedTokenIndex].token,
+        new PublicKey(receiver),
+        amount,
+        tokens[selectedTokenIndex].laoId,
+      ).then(
+        () => toast.show('Sent transaction'),
+        (reason) => {
+          toast.show('Failed to send transaction');
```

Comment:
It could be nice to add all these toast message strings in `string.ts` (because they are display in the UI).